README.md

```diff
@@ -57,27 +57,26 @@ run.
 
 ### Aggregator Address
 
-The aggregator is run and managed by the Oak team. This address will be hard-coded.
-perator.
+The aggregator is run and managed by the Ava Protocol team. This address will be hard-coded.
 
 #### Holesky Testnet
 
-- https://aggregator-holesky.api.oak.tech
+- https://aggregator-holesky.api.avaprotocol.org
 
 #### Mainnet
 
-- https://aggregator.api.oak.tech
+- https://aggregator.api.avaprotocol.org
```

Comment:
@v9n could we use avaprotocol.org domain name for all new endpoints going forward? We need to update these two.